### DISCUSSION TITLE
does your support exist for paying customers?

### DISCUSSION THREAD

**Discussion by FranziskaSom** [rustdesk/rustdesk#7057] [General]
we have the pro version of a self-hosted server. we wrote to you about 1 week ago. nothing. empty. black hole. no answer. yawning emptiness. no confirmation that the ticket has been received. no answer on discord. professionalism really looks different ...

**Comment by rustdesk** [comment]
Apologize for this. Would you write to support@rustdesk.com? or post on https://github.com/rustdesk/rustdesk-server-pro/issues , usually we can respond very quickly if the description is clear. Discord is maintained by our contributor.
